### PR TITLE
1.24 Exception File

### PR DESCRIPTION
**Pull request by gracenng**
/kind documentation

Recording 2 exceptions for 1.24 enhancement freeze on February 3rd

### CODE REVIEW

File: releases/release-1.24/exceptions.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Release Team Lead: James Laverack ([@JamesLaverack](https://github.com/JamesLaverack) 
+
+# Enhancements Freeze Exceptions requested in 1.24
+- name: "Add support for resizing PVs"
+  issue: 284
+  date_requested: 2022-02-03
+  date_reviewed: 2022-02-03
+  thread: "https://groups.google.com/a/kubernetes.io/g/release-team/c/x6AlKFEQU4A"
+  pull_requests:
+    - "https://github.com/kubernetes/enhancements/pull/284"
```
Comment (JamesLaverack) [inline releases/release-1.24/exceptions.yaml:12]:
```suggestion
    - "https://github.com/kubernetes/enhancements/pull/3210"
```
This should be the PR that introduced the fix.

File: releases/release-1.24/exceptions.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Exception requests in 1.24
+# Google Group: https://groups.google.com/forum/#!forum/kubernetes-sig-release
+# Release Team Lead: James Laverack ([@JamesLaverack](https://github.com/JamesLaverack) 
+
+# Enhancements Freeze Exceptions requested in 1.24
+- name: "Add support for resizing PVs"
+  issue: 284
+  date_requested: 2022-02-03
+  date_reviewed: 2022-02-03
```
Comment (JamesLaverack) [inline releases/release-1.24/exceptions.yaml:9]: Looking at the mailing list it was filed at 2022-02-03 03:35:21 UTC. So either the 3rd or the 4th depending on your timezone. We gave the approval on 2022-02-04 17:19:58 UTC, so I'd say that's the 4th for sure.

File: releases/release-1.24/exceptions.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    - "https://github.com/kubernetes/enhancements/pull/284"
+  status: "approved"
+  
+  - name: "AdminNetworkPolicy support"
+  issue: 2091
+  date_requested: 2022-02-04
+  date_reviewed: 2022-02-04
+  thread: "https://groups.google.com/a/kubernetes.io/g/release-team/c/SiJdoltT_o8/m/mEuwygA8CwAJ"
+  pull_requests:
+    - "https://github.com/kubernetes/enhancements/pull/2091"
```
Comment (JamesLaverack) [inline releases/release-1.24/exceptions.yaml:21]:
```suggestion
    - "https://github.com/kubernetes/enhancements/pull/3212"
```

**Comment by gracenng** [comment]
Apologies I read the github links as issues. Thanks for the feedback @JamesLaverack, updated

**Review by sftim — COMMENTED** [review]
Should this merge _before_ the v1.24 release, or afterwards?

**Comment by gracenng** [comment]
I still have the code freeze exceptions to add 🤓 Will do that asap if we
need to merge before

**Review by saschagrunert — APPROVED** [review]

**Comment by saschagrunert** [comment]
/hold

If @gracenng still wants to add something.

**Comment by gracenng** [comment]
I did a little clowning and accidentally remove all the changes here 🤡 

All the information is available in the Google Group so I'll start over and dig them up 🥴

Apologies, this should have been merged already :((

**Comment by sftim** [comment]
@gracenng maybe https://github.com/kubernetes/sig-release/compare/0de2acb3a7560d9ffddeb37dd8d6c9bf21a5f1b2..36677c773bc27e93c15aa0eb8910210c6eb5614d is what you were looking for?

**Pull request closed without merging**